src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.debezium.converters;
+
+import java.util.Properties;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+import io.aiven.kafka.connect.debezium.converters.utils.DummyRelationalColumn;
+import io.aiven.kafka.connect.debezium.converters.utils.MoneyRelationalColumn;
+
+import io.debezium.spi.converter.CustomConverter;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class MoneyConverterTest {
```

Comment:
What about the case when the column is optional? Should not we also cover it?

Reply:
Sure!